FrontEnd/main/src/App.js

```diff
@@ -17,6 +20,12 @@ function App() {
             <MainCompanies/>
             <MainLoginBanner/>
             <MainAboutSection/>
+            <div className="App">
```

Comment:
is the className="App" used somewhere? If not, I think it can be removed

Reply:
Thank you. Done